base/src/main/java/io/spine/type/KnownTypes.java

```diff
@@ -45,15 +47,34 @@
  * @author Dmytro Dashenkov
  */
 @Internal
-public class KnownTypes {
+public enum KnownTypes {
 
-    private final TypeSet types;
+    INSTANCE;
+
+    @SuppressWarnings({
+            "TransientFieldNotInitialized", // We substitute the instance in readResolve()
```

Comment:
As long as you have period at the end of the following sentence, let's have a period here as well.